dispatcher/DAQController.py

```diff
@@ -12,12 +13,16 @@
 aggregated status updates and commands from the mongo connector and decides if
 any action needs to be taken to get the DAQ into the target state. It also handles the
 resetting of runs (the ~hourly stop/start) during normal operations.
-'''
+"""
+
 
 def now():
     return datetime.datetime.now(pytz.utc)
     #return datetime.datetime.utcnow() # wrong?
 
+# Communicate between various parts of dispatcher that no new run was determined
+NO_NEW_RUN = -1
```

Comment:
Why redefine here rather than import from the other file?

Reply:
valid